pkg/model/iam.go

```diff
@@ -64,33 +68,63 @@ func (b *IAMModelBuilder) Build(c *fi.ModelBuilderContext) error {
 	// Generate IAM objects etc for each role
 	for _, role := range roles {
 		name := b.IAMName(role)
+		roleAsString := reflect.ValueOf(role).String()
 
 		var iamRole *awstasks.IAMRole
-		{
-			rolePolicy, err := b.buildAWSIAMRolePolicy()
-			if err != nil {
-				return err
+
+		arn := ""
+
+		if b.Cluster.Spec.CustomPolicies != nil && featureflag.CustomPoliciesSupport.Enabled() {
```

Comment:
If the feature flag isn't set, this should be an error, not silently ignored.  Probably in validation.

This may not need a feature flag.